uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/utils/DirectionUtils.kt

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.uikit.viewmodels.common.utils
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.sygic.maps.uikit.viewmodels.R
+import com.sygic.sdk.route.RouteManeuver
+
+object DirectionUtils {
+
+    fun isManeuverRoundabout(maneuver: RouteManeuver?): Boolean = maneuver?.let {
```

Comment:
nebola by toto dobra extensna `RouteManeuver.isRoundabout()`?

Reply:
vlastne aj tie ostatne... pri signposte to mas ako extension func

Reply:
ano, u sebe na locale mám už ten isRoundabout jako extension. mohu přehodit i ty zbylé, díky.